src/Rules/Spider.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
I think this is overkill. We know that UAParser simply classify all bots as *spider*, which is why it was just adding a simple rule originally. Now it's iterating arrays that we never expect to change.